src/Catalyst.TestUtils/TransactionHelper.cs

```diff
@@ -0,0 +1,71 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using Catalyst.Core.Lib.Extensions;
+using Catalyst.Protocol.Cryptography;
+using Catalyst.Protocol.Network;
+using Catalyst.Protocol.Transaction;
+using Catalyst.Protocol.Wire;
+using Google.Protobuf.WellKnownTypes;
+using Nethermind.Dirichlet.Numerics;
+
+namespace Catalyst.TestUtils
+{
+    public static class TransactionHelper
+    {
+        public static TransactionBroadcast GetPublicTransaction(uint amount = 123,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method has 8 parameters, which is greater than the 7 authorized.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4318201)